party_time/party_time.lua

```diff
@@ -313,6 +389,62 @@ ui.display(function()
         end
     end
 
+    -- Dialog for Selecting new Party|Alliance Leader
+    if unhandled_leader then
+        local height = 36
+        for index, member in pairs(party) do
+            if type(index) == "number" then
+                if index <= 6 and member.id ~= player.id then
+                    height = height + 24
+                elseif party.alliance.alliance_leader_id == player.id and (member.id == party.alliance.party_2_leader_id or member.id == party.alliance.party_3_leader_id) then
+                    height = height + 24
```

Comment:
This `elseif` case is the same as the `if` case before? Is that meant to be the case? If so, would merge them.